rasa/nlu/test.py

```diff
@@ -506,6 +490,40 @@ def evaluate_intents(
     }
 
 
+def try_plot_confusion_matrix(
+    output_directory: Optional[Text],
+    confmat_filename: Optional[Text],
+    target_intents: List[IntentEvaluationResult],
+    predicted_intents: List[ResponseSelectionEvaluationResult],
+):
+    from sklearn.metrics import confusion_matrix
+    from sklearn.utils.multiclass import unique_labels
+
+    if output_directory:
+        confmat_filename = os.path.join(output_directory, confmat_filename)
+
+    cnf_matrix = confusion_matrix(target_intents, predicted_intents)
+    labels = unique_labels(target_intents, predicted_intents)
+    plot_confusion_matrix(
+        cnf_matrix,
+        classes=labels,
+        title="Intent Confusion matrix",
+        out=confmat_filename,
+    )
+
+
+def try_plot_hostogram(
+    output_directory: Optional[Text],
+    intent_hist_filename: Optional[Text],
+    intent_results: List[IntentEvaluationResult],
+):
+    if output_directory:
```

Comment:
we could check that further up and rename to `_plot_histogram` ?

Reply:
same here